pkg/utils/common.go

```diff
@@ -128,6 +128,18 @@ var (
 		Resource: placementv1beta1.ClusterResourcePlacementResource,
 	}
 
+	ClusterRoleGVK = schema.GroupVersionKind{
```

Comment:
can you move these definition to your test file or test/util package? these are not fleet related resources.

Prefer not to mix the test code with production code.